ops/test/swap/swap.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+    required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = "~>2.0"
+    }
+  }
+}
+
+provider "azurerm" { 
+    features {}
+    skip_provider_registration = true # but why?
+}
+
+resource "azurerm_app_service_active_slot" "promote_staging" {
+  resource_group_name   = "prime-simple-report-test"
+  app_service_name      = "simple-report-api-test"
+  app_service_slot_name = "staging"
+}
```

Comment:
This is quite the weird Terraform resource, but I understand what it's doing.

After this is deployed, does the current production slot become the staging slot? If not, does the staging slot disappear and require a redeploy?

Reply:
The production slot goes to staging.

Reply:
For the actual swap, do we need to do any prep work on the staging slot like warming up the app? When/how do we know it's okay to flip the switch once we deploy something to the staging slot?